app/models/concerns/titleable.rb

```diff
@@ -19,7 +19,7 @@ def canonical_title_key
   private
 
   def has_english_title?
-    titles.keys.any? { |k| k.end_with?('_en') || k.start_with?('en_') }
+    titles.keys.any? { |k| k.start_with?('en') }
```

Comment:
why did we get rid of `end_with?('_en')`, we should still be doing that.

Reply:
Why would we?

Reply:
### Anime

Found 0 entries on page 837, 16740 checked
Total 0 entries

### Manga

Found 0 entries on page 2778, 55560 checked
Total 0 entries